src/ensembl/src/content/app/entity-viewer/components/feature-length-axis/featureLengthAxisHelper.ts

```diff
@@ -0,0 +1,58 @@
+import { ScaleLinear } from 'd3';
+
+export const getTicks = (scale: ScaleLinear<number, number>) => {
+  // use d3 scale to get 'approximately' 10 ticks (exact number not guaranteed)
+  // which are "human-readable" (i.e. are multiples of powers of 10)
+  // and are guaranteed to fall within the scale's domain
+  let ticks = scale.ticks();
+  const length = scale.domain()[1]; // get back the initial length value on which the scale is based
+
+  // choose only the "important" ticks for labelling
+  const exponent = Number(
+    (length.toExponential().match(/e\+(\d+)/) as string[])[1]
+  );
+  const powerOfTen = 10 ** exponent; // e.g. 100, 1000, 10000, etc.
+  ticks = ticks.filter((number) => {
+    // do not add a tick in the end of the ruler (it is handled specially)
+    // and throw away all the possible 'inelegant' intermediate ticks, such as 50, etc.
+    return number !== length && number % powerOfTen === 0;
```

Comment:
Since we are filtering out ticks such as 50:
For length 500:
![Screenshot 2020-01-30 at 12 12 44](https://user-images.githubusercontent.com/3085815/73448994-149a5e00-435a-11ea-9a37-3ceffa62ce8e.png)


For length 1000:
![Screenshot 2020-01-30 at 12 13 06](https://user-images.githubusercontent.com/3085815/73448997-17954e80-435a-11ea-8740-fc33752522f3.png)

Should the one with length 1000 be having more ticks?

Reply:
Updated the code so that if the length of feature falls in the the interval between a power of ten and a power of ten plus distance between two ticks (in this case, 1000-1100), the ruler shows intermediate unlabelled ticks as well.